src/service.rs

```diff
@@ -523,6 +523,12 @@ impl<P: Platform> ServiceResources<P> {
                 let previous_status = self.platform.user_interface().status();
                 self.platform.user_interface().set_status(ui::Status::WaitingForUserPresence);
                 loop {
+                    if ctx.interrupt.map(|i|i.is_interrupted()) == Some(true) {
+                        // Error does not matter as it will be dropped anyway
+                        info_now!("User presence request cancelled");
+                        return Ok(reply::RequestUserConsent{result: Err(consent::Error::Interrupted)}.into());
+                    }
```

Comment:
nit: formatting is off here (missing spaces)

Reply:
Weird, rustfmt does not format this part.

running with ` --config error_on_unformatted=true` gives

```
rustfmt --edition 2018 src/service.rs --config error_on_unformatted=true
error[internal]: not formatted because a comment would be lost
   --> /home/soso-nitrokey/Code/trussed/src/service.rs:149
    |
149 |         match request {
    |
    = note: set `error_on_unformatted = false` to suppress the warning against comments or string literals

warning: rustfmt has failed to format. See previous 1 errors.
```

I'll manually fix this formatting, but if we remove the offending comment, it changes the entire `match` statement, so it would make a large diff.

It should probably be done in another PR.